lib/components/BuilderUI.jsx

```diff
@@ -74,7 +79,7 @@ const sendPrizeStrategyTx = async (
   const singleRandomWinnerConfig = {
     rngService: rngServiceAddress,
     prizePeriodStart: prizePeriodStartTimestamp,
-    prizePeriodSeconds,
+    prizePeriodSeconds: prizePeriodInSeconds,
```

Comment:
How would you feel about renaming `prizePeriodInSeconds` to `prizePeriodSeconds` so we can skip this renaming?